lib/teiserver/game/servers/balancer_server.ex

```diff
@@ -162,8 +206,10 @@ defmodule Teiserver.Game.BalancerServer do
     |> Base.encode64()
   end
 
+  # This function is public only for testing; it should not be called otherwise
+  # long term, there in interest in confirming this is a stateless pure function, which would make this easier to test
```

Comment:
If I get the stateful tests working at all, it should be a snap to go back and write one or two tests of this pure function.